src/jwks.cc

```diff
@@ -73,6 +76,18 @@ class EvpPkeyGetter : public WithStatus {
     return createEvpPkeyFromRsa(rsa.get());
   }
 
+  bssl::UniquePtr<EVP_PKEY> createEvpPkeyFromPkcs8(
+      const std::string& pkey_pem) {
+    bssl::UniquePtr<BIO> buf(BIO_new_mem_buf(pkey_pem.data(), pkey_pem.size()));
```

Comment:
is it possible that this could return null?.  If so, need to check it and return error status

Reply:
Based on https://www.openssl.org/docs/man1.0.2/man3/BIO_new_mem_buf.html I don't *think* so, but I'm not fully confident so I'll add a check. The OpenSSL C++ API is kind of like a sword with no handle